### DISCUSSION TITLE
¿Como encontraron Launch X ?

### DISCUSSION THREAD

**Discussion by Mar-rbw** [Launch-X-Latam/MisionFrontEnd#69] [General]
Buenos días, buenas tardes ¿Cómo están?
Soy su compañero Marcelo, un estudiante cualquiera.
El querer interactuar con ustedes  y una pisca de curiosidad, quiero saber de ustedes.

**¿Cómo encontraron Launch X y que los motivo a unirse?**

En mi caso, lo encontré de forma ociosa viendo cortos de YouTube hasta llegar al canal "No es Brujería, es Tecnología - brujeriatech" , donde es mencionado el tema. Atrapando mi interés de forma inmediata ,pues vi una posibilidad de relacionarme con otros y poder tener la posibilidad de desarrollarme como individuo, a la vez, como futuro profesional.

**Comment by JosafatJimenezB** [comment]
En mi caso primero entre a innovaccion virtual por medio de un anuncio en facebook donde invitaban a los alumnos de una universidad a aprender sobre la nube, a finales del año anterior dieron a conocer que crearían un bootcamp fullstack como nuevo programa y asi entre a launchx primera edicion, despues por medio de las redes sociales dieron a conocer que crearian otro pero version latam y pues aqui estamos jajaja

**Reply by JoseAnayaR** [reply]
Vengaa!! Yo tambien estuve en la edicion pasada, cuando vi que habria otra no lo pense 2 veces en inscribirme y poder retroalimentar lo ya aprendido y la primera ves me lo recomendo una amiga que ya estaba inscrita y sabia de el interes que tengo por aprender de este tema!

**Reply by JosafatJimenezB** [reply]
orales que chido encontrar antiguos explorer, y que backend escogiste?

**Comment by descob4r** [comment]
Estaba viendo tiktoks y en eso me salió uno de No es brujería, es tecnología... me uni el mes pasado al az-500 e investigando más de innovacción llegue aquí, por que quiero hacer varios proyectos tanto personales y para la empresa en la que estoy.

**Reply by albertt206** [reply]
Yo llegue de la misma forma por un video de Tiktok de No es brujeria es tecnologia. Saludos desde Peru

**Reply by rromi182** [reply]
Yo también llegué de la misma manera, apenas lo vi y me inscribí, vi como una oportunidad más de poder aprender, a decir verdad no entendía mucho del programa y hasta me olvidé de ello hasta el día que me llegó el dichoso correo de Launch X xd, a partir de ahi me puse a investigar más sobre el programa y aqui estamos! Saludos desde Paraguay 👩‍💻.

**Reply by Melissabasulto18** [reply]
Yo igual vi la información por medio de TikTok, se me hizo muy interesante por lo cual me inscribí y aquí estamos 😊

**Reply by OscarHub90** [reply]
Hola, al igual que ustedes vi el anuncio pero en historias de Instagram de brujeriatech..bendito canal 👍

**Reply by IngCarrillo12** [reply]
Igualmente vi la informacion por tiktok

**Reply by descob4r** [reply]
Si al principio no creía por que faltaba información en la página y cuando me suscribí no llego nada de correo, como de espera o algo así

Tuve el pensamiento de pago web... nada es gratis jajaja pero ya vimos que innovaccion sí xD

**Comment by mariel-rs** [comment]
En mi caso, encontré LaunchX por un post en Linkedin. Front-end es una asignatura pendiente, entonces aquí vamos 🤪

**Comment by HenocGnzlez** [comment]
Yo lo encontré porque un compañero me paso un link para cursos de ciberseguridad que da Microsoft, si no mal recuerdo era Operación Aleph pero se había cerrado la convocatoria entonces seguí buscando cuando abrían otro y encontré LaunchX

**Comment by GusAlhazred** [comment]
De la misma manera. Vengo estudiando de autodidacta hace 1 mes, mas o menos. Y en un momento de relax, estaba mirando shorts en YouTube y aparecio el short de brujeriatech.
Me acuerdo que me levante y lo hice en el momento haha

**Comment by JocelinFlores** [comment]
A traves de FB observe una publicación sobre los cursos por parte de innovaccion y microsoft y me inscrito a la mayoria de cursos que han dado (como niño que se forma a pedir dulces una y otra vez  ) y aqui estamos de nuevo.

**Comment by koshishark** [comment]
A través de TikTok, creo que de Brujería Tech o de PacoWeb.

**Comment by cavazte** [comment]
Como muchos, igual llegué por medio de un tiktok de Brujería Tech

**Comment by ByLobs** [comment]
en mi caso me lo recomendó un amigo

**Comment by Ignacioskm** [comment]
En mi caso encontré a Launch X  a través de un tiktoker de programación.

**Comment by LaMazorca** [comment]
Al igual que tú llegue viendo al tiktoker brujería tech, pero yo ví el anterior, y no pude ingresar porque pedían ser estudiante o egresado en alguna rama de tecnología, en mi caso, lo que yo estoy haciendo es cambiar de carrera (soy paramédico), y pues seguí las redes sociales e innovación por si se da una oportunidad. De hecho, ya me inscribí a la universidad para estudiar ingeniería en sistema de software, en enero comienza. 😄😄😃🥰

**Reply by ByLobs** [reply]
Que tengas éxito en tus estudios 👍🏻

**Reply by LaMazorca** [reply]
Gracias, hago el mejor esfuerzo y aprovecho aprender de todos ustedes aqui

**Comment by Ketote95** [comment]
Creo que de la misma manera que la mayoría aquí 🤣 Osea, por curiosidad.
Soy una persona curiosa, actualmente estoy en lo último de mi carrera en la universidad que es Ingeniería de Sistemas. Me gusta muchísimo y siempre estoy en busca de nuevos retos :3
Hubo un año en el que mi equipo y yo, fuimos ganadores de un evento en mi país llamado HackBo 2019, que consistía en crear algo innovador de acuerdo a la categoría. 24 horas seguidas de pura programación ♥
Vi una convocatoria de Launch X gracias al estado WhatsApp de un amigo, y pues, me llamó bastante la atención 👀 Bueno, y aquí me tienen haha

**Reply by descob4r** [reply]
Y cual fue la creación con la que ganaron ?

**Comment by samujm** [comment]
A mi me mandó la liga de registro un amigo y pues me dí la oportunidad de inscribirme.

**Comment by RafaelMartinCallataFlores** [comment]
En mi caso, también me entere por el canal "No es Brujería, es Tecnología - brujeriatech" , donde es mencionado el tema. y cuando vi los detalles a través del portal de innovacion no dude y aproveche la oportunidad de aprender y conocer a mas personas en el mismo rubro

**Comment by ghost** [comment]
no se pero me llego el correo con toda la info y no dude. Muchos compañeros se inscriben en las mismas cosas que yo pero no han sido tan afortunados essta vez jeje

**Comment by MaferMtz** [comment]
Yo entre al de innovaccion virtual por un amigo que me lo recomendo, me gusto muchisimo y me entere de esta edicion por la cuenta de brujeria tech y dije tengo que aprovechar, aqui andamos a pesar de que es un gran esfuerzo para todos los que trabajamoss o estudiamos dedicarle un poco mas de tiempo cuando en realidad quieres descansar, pero vale mucho la pena
